src/AggregateRoot.php

```diff
@@ -51,50 +29,4 @@ protected function __construct()
     }
 
     abstract protected function aggregateId(): string;
```

Comment:
oh was late yesterday. Need to revert my comment. A trait can contain abstract methods ...